ngraph/frontend/frontend_manager/CMakeLists.txt

```diff
@@ -33,3 +33,21 @@ target_include_directories(${TARGET_NAME} PUBLIC $<BUILD_INTERFACE:${FRONTEND_IN
 target_include_directories(${TARGET_NAME} PRIVATE ${NGRAPH_INCLUDE_PATH} ${FRONTEND_INCLUDE_DIR})
 
 target_include_directories(${TARGET_NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
+
+# Installation rules
+install(TARGETS ${TARGET_NAME} EXPORT ngraphTargets
+        RUNTIME DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph
+        ARCHIVE DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph
+        LIBRARY DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph)
+
+install(DIRECTORY ${FRONTEND_INCLUDE_DIR}/frontend_manager
+    DESTINATION ${FRONTEND_INSTALL_INCLUDE}
+    COMPONENT ngraph
+    FILES_MATCHING
+        PATTERN "*.hpp"
+        PATTERN "*.h"
+)
```

Comment:
FYI: if we want this to be merged before CF, we need to comment this code otherwise it will be a part of the release package.

Reply:
We're not expecting to merge it before CF.